docs/cloud/index.md

```diff
@@ -0,0 +1,18 @@
+# DAOS in the Cloud
+
+In addition to on-prem deployments, DAOS is available as part of
+cloud offerings.
```

Comment:
DAOS can be installed in GCP and we have Terrform modules and examples that show how to deploy it. But DAOS is not an official service offering on GCP. 

We should probably change the wording here so that it doesn't lead someone to believe that it's an official service offering or a GCP Marketplace offering.  "DAOS is available as part of cloud offerings" may lead someone to think that it is available in the GCP Marketplace.